src/sentry/models/sentryfunction.py

```diff
@@ -25,7 +26,7 @@ class SentryFunction(DefaultFieldsModel):
 
     organization = FlexibleForeignKey("sentry.Organization")
     name = models.TextField()
-    slug = models.CharField(max_length=64, unique=True)
+    slug = SentrySlugField(max_length=64, unique=True, db_index=False)
```

Comment:
Same thing, should this include `db_index=False`?

Reply:
Yup! Take a look at my original PR comment: https://github.com/getsentry/sentry/pull/62246#issue-2053195086, but also, you can see from the CI that no real migration is happening, meaning the values are "correct" and consistent with their base values.